install_bootstrap

```diff
@@ -241,14 +245,21 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
         discourse_category_id='discourse_category_id'
         ask_question "What is your Discourse category id (9 for staging, 6 for production)?" "$DISCOURSE_CATEGORY_ID" "discourse_category_id" $discourse_category_id
 
+        zenodo_client_id='zenodo_client_id'
+        ask_question "What is your zenodo client id?" "$ZENODO_CLIENT_ID" "zenodo_client_id" $zenodo_client_id
+        zenodo_client_secret='zenodo_client_secret'
+        ask_question "What is your zenodo client secret?" "$ZENODO_CLIENT_SECRET" "zenodo_client_secret" $zenodo_client_secret
+        zenodo_url='zenodo_url'
+        ask_question "What is the zenodo url?" "$ZENODO_URL" "zenodo_url" $zenodo_url
```

Comment:
Might want to give the example urls, https://sandbox.zenodo.org (for staging) and https://zenodo.org, so that we don't accidentally point to prod zenodo from staging.